admin_guide/managing_networking.adoc

```diff
@@ -872,6 +879,81 @@ is complicated, because nodes have both "external" IP addresses and
 bypass it.
 
+[[enabling-fixed-ips-for-external-project-traffic]]
+== Enabling Fixed IPs for External Project Traffic 
+
+As a cluster administrator, you can assign specific, fixed IP addresses to
+projects, so that traffic is externally easily recognizable. This is different
+from the default egress router, which is used to send traffic to specific
+destinations.
+
+Recognizable IP traffic increases cluster security by making the origins
+visible. Once enabled, all outgoing external connections from the specified
+project will share the same, fixed source IP, meaning that any external
+resources can recognize the traffic.
+
+Unlike the egress router, this is subject to `EgressNetworkPolicy` firewall
+rules.
+
+[IMPORTANT]
+====
+Enabling fixed IPs for external project traffic is a Technology Preview feature
+only. 
+ifdef::openshift-enterprise[] 
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+To enable fixed source IPs:
+
+. Update the `NetNamespace` with the desired IP:
++
+----
+oc patch netnamespace <project_name> -p '{"egressIPs": ["<IP_address>"]}'
+----
++
+For example, to assign the MyProject project to an IP address of 192.168.1.100:
++
+----
+oc patch netnamespace MyProject -p '{"egressIPs": ["192.168.1.100"]}'
+----
++
+The `egressIPs` field is an array, but must be set to a single IP address. If setting multiple IPs, the other IPs will be ignored.
+
+//The following steps can maybe be deleted when tech preview is over when this process if automatic:
+. Manually assign the egress IP to the desired node hosts. Set the `egressIPs` field on the `HostSubnet` object on the node host. Include as many IPs as you want to assign to that node host:
++
+----
+oc patch hostsubnet <node_name> -p \
+  '{"egressIPs": ["<IP_address_1>", "<IP_address_2>"]}'
+----
++
+For example, to say that node1 should have the egress IPs 192.168.1.100, 192.168.1.101, and 192.168.1.102:
++
+----
+oc patch hostsubnet node1 -p \
+  '{"egressIPs": ["192.168.1.100", "192.168.1.101", "192.168.1.102"]}'
+----
++
+[IMPORTANT]
+====
+Egress IPs are implemented as additional IP addresses on the primary network
+interface, and must be in the same subnet as the node's primary IP. Allowing additional IP addresses on the primary network interface might require extra configuration when using some cloud or VM solutions.
+====
+
+If the above is enabled for a project, all egress traffic from that project will
+be routed to the node hosting that egress IP, then NATted to that IP address. If
```

Comment:
NATted. Should this be NAT'd? 
(Natted in the urban dictionary is a very different thing! http://www.urbandictionary.com/define.php?term=natted )

Reply:
Hmm right. Replaced with "connected (using NAT)"